controller/src/test/java/io/pravega/controller/server/eventProcessor/ControllerEventProcessorsTest.java

```diff
@@ -106,21 +105,30 @@ public void testHandleOrphaned() throws CheckpointStoreException {
         EventProcessorSystem system = mock(EventProcessorSystem.class);
         EventProcessorGroup<ControllerEvent> processor = getProcessor();
         EventProcessorGroup<ControllerEvent> mockProcessor = spy(processor);
+        CuratorZookeeperClient curatorZKClientMock = mock(CuratorZookeeperClient.class);
```

Comment:
This mock object is created but not returned by CuratorClient object used by CheckpointStore. So how is this helping?

Reply:
done